ui/src/app/components/admin/audit-card-list-view.tsx

```diff
@@ -0,0 +1,131 @@
+import colors from 'app/styles/colors';
+import * as fp from 'lodash/fp';
+import * as moment from 'moment';
+import * as React from 'react';
+import {
+  AuditAction,
+  AuditEventBundle,
+  AuditEventBundleHeader,
+  AuditTargetPropertyChange
+} from '../../../generated';
+import {ActionAuditCardBase} from '../card';
+import {FlexRow} from '../flex';
+
+
+const HidableCell = (props: {content: string}) => {
+  const {content} = props;
+  return <div style={{backgroundColor: content ? 'white' : '#f0f3f5',
+    border: '1px solid',
+    boxSizing: 'border-box'}}>{content}</div>;
+};
+
+const PropertyChangeListEntry = (props: {targetProperty?: string, previousValue?: string, newValue?: string}) => {
+  const {targetProperty, previousValue, newValue} = props;
+  return <React.Fragment>
+    <HidableCell content={targetProperty}/>
+    <HidableCell content={previousValue}/>
+    <HidableCell content={newValue}/>
+  </React.Fragment>;
+};
+
+const PropertyChangeListView = (props: { propertyChanges: AuditTargetPropertyChange[] }) => {
+  const {propertyChanges} = props;
+  const propertyCellStyle = {fontWeight: 600, border: `1px solid`};
+
+  return propertyChanges.length > 0 ?
+  <div style={{
+    margin: '0.25rem 0 0rem 1rem',
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)'
+  }}>
+    <div style={propertyCellStyle}>Changed Property</div>
+    <div style={propertyCellStyle}>Previous Value</div>
+    <div style={propertyCellStyle}>New Value</div>
+    {propertyChanges.map((propertyChange, index) => <PropertyChangeListEntry {...propertyChange} key={index}/>)}
+  </div> : <div style={{margin: '0.25rem 0 0rem 1rem', fontStyle: 'italic'}}>No Property Changes</div>;
+};
+
+const typeTextColor   = colors.primary;
```

Comment:
just use the colors directly.

Reply:
That makes sense, but now I'm tempted to break out two new named fragments so it's obvious what's what. It's not the most intuitive data structure to work with.